core/services/keystore/keys/ocr2key/generic_key_bundle.go

```diff
@@ -41,6 +42,10 @@ var _ KeyBundle = &keyBundle[*solanaKeyring]{}
 var _ KeyBundle = &keyBundle[*terraKeyring]{}
 var _ KeyBundle = &keyBundle[*starknetKeyring]{}
 
+func emptyKeyBundle[K keyring](key K) *keyBundle[K] {
```

Comment:
I would have used the name `newKeyBundle`. Maybe that variant should be named `newRandomKeyBundle`?

Reply:
I like either `emptyKeyBundle` or `newEmptyKeyBundle` to emphasize that it's empty versus `random` feels like there is already an initialized privatekey value?

Reply:
I meant that I didn't use `newKeyBundle` because it was already taken by something that is doing random intitialization. If we renamed that one to `newRandomKeyBundle`, then this could be `newKeyBundle`.

Reply:
Updated.